package.json

```diff
@@ -17,12 +17,14 @@
         "axe-core": "^4.8.3",
         "cypress": "^13.6.1",
         "cypress-axe": "^1.5.0",
+        "node-fetch": "^3.3.2",
         "webpack-merge": "^5.8.0"
     },
     "scripts": {
         "build": "wp-scripts build ./src/onboarding.js ./src/Scripts/sitegen-theme-marker/sitegen-theme-marker.js ./src/Scripts/sitegen-theme-marker/sitegen-theme-marker.css",
         "format": "wp-scripts format ./src",
         "start": "wp-scripts start ./src/onboarding.js ./src/Scripts/sitegen-theme-marker/sitegen-theme-marker.js ./src/Scripts/sitegen-theme-marker/sitegen-theme-marker.css",
+        "set-version-bump": "node ./.github/scripts/set-version-bump.js && npm i && rm -rf ./build && npm run build && composer run i18n",
```

Comment:
Should we add `composer run i18n-php` and `composer run i18n-json` as well?

Reply:
This command wraps all the i18n commands into one. See https://github.com/newfold-labs/wp-module-onboarding/blob/trunk/composer.json#L68C3-L74C1 it starts with `make-pot`, then `update-po`, `make-php` and `make-json`.